src/main/kotlin/domain/Board.kt

```diff
@@ -0,0 +1,18 @@
+package domain
+
+class Board(width: Int, height: Int, private val generator: MineGenerator) {
+
+    val board: Map<Dot, DotState>
+
+    init {
+        board = (1..width).map {
+            createDot(it, height)
+        }.flatten().toMap()
+    }
+
+    private fun createDot(width: Int, height: Int): List<Pair<Dot, DotState>> {
```

Comment:
```suggestion
    private fun createDot(width: Int, height: Int): List<Map<Dot, DotState>> {
```
Pair대신 Map을 활용해보면 어떨까요?
굳이 Pair을 받아서 Map으로 변환하지 않고, 바로 Map을 사용할 수 있을 것 같아요